docs/en/04_Changelogs/4.0.0.md

```diff
@@ -109,6 +242,107 @@ Core template locations have moved - if you're including or overriding these
 no longer exists, and instead template locations will be placed in paths that match
 the `SilverStripe\Forms` namespace.
 
+#### Upgrade your member statics
```

Comment:
`member` implies `Member` dataobject; Misleading title.

How about `Upgrade class config static properties to private`?